src/App.js

```diff
@@ -0,0 +1,29 @@
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
```

Comment:
This function should be refactored to ES6 syntax. So it must be `const App = () => {...}`